src/main/java/Duke.java

```diff
@@ -1,10 +1,254 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+    private final static String LOAD_PATH = "save/tasks.txt";
+
+    private static void addTask(ArrayList<Task> tasks, Task task) {
+        tasks.add(task);
+        System.out.println("Got it. I've added this task:");
+        System.out.println(task);
+        System.out.printf("Now you have %d tasks in the list\n", tasks.size());
+    }
+
+    private static void doMark(ArrayList<Task> tasks, String[] wordList) throws DukeException {
+        if (wordList.length != 2) {
+            throw new DukeException("☹ OOPS!!! Wrong number of arguments for mark");
```

Comment:
Consider avoiding the usage of magic literals in the code. [See here](https://nus-cs2113-ay2223s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-magic-numbers)